dogs/src/app/shared/api.service.ts

```diff
@@ -21,4 +21,8 @@ export class ApiService {
 		return this.http.get(`https://dog.ceo/api/breed/${dogName}/list`);
 	}
 
+	public getSubBreedImageURL(dogName: string, subBreedName: string): Observable<Object> {
+		return this.http.get(`https://dog.ceo/api/breed/${dogName}/${subBreedName}/images/random`);

```

Comment:
you can store the common part of the URL intro a const
for example: "https://dog.ceo/api/"

Reply:
and call the const every time you need it